### PR TITLE
zh-translation: /docs/setup/install/istioctl/index.md

### PR DESCRIPTION
**Pull request by helight**
ref:https://github.com/servicemesher/istio-official-translation/issues/1726

### CODE REVIEW

File: content/zh/docs/setup/install/istioctl/index.md [hunk 1]
```diff
@@ -235,7 +235,7 @@ $ istioctl verify-install -f $HOME/generated-manifest.yaml
 可以使用命令上的 `--set` 选项分别设置此 API 中的配置参数。 例如，要在默认配置文件中禁用遥测功能，请使用以下命令：
 
 {{< text bash >}}
-$ istioctl manifest apply --set values.global.mtls.enabled=true --set values.global.controlPlaneSecurityEnabled=true
+$ istioctl manifest apply --set values.global.mtls.enabled=true
 {{< /text >}}
 
 Alternatively, the `IstioControlPlane` configuration can be specified in a YAML file and passed to
```
Comment (yuxiaobo96) [inline content/zh/docs/setup/install/istioctl/index.md]: 此处英文需要翻译（原文中是没有翻译的）

Comment (helight) [reply content/zh/docs/setup/install/istioctl/index.md]: accept

**Pull request merged**